### ISSUE TITLE
TreatmentCitation QC correction

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#591]
we need to look into automating wrong  or missing `treatmentCitation `and especially adding the attributes.

`treatmentCitation` needs to year, author, page

`bibRefCitation` needs all the respective attributes

I could imaging this could work similar to the other parse tools we already have

**Comment by gsautter** [comment]
I agree. We can do something like "Mark Bibliographic Citation" akin to "Mark Figure Citation".

On top of that, in a selection including both a `taxonomicName` and a `bibRefCitation`, we can add something like "Mark Treatment Citation" or "Mark Treatment Citation Group" (the latter for multiple `bibRefCitation`s).

**Comment by myrmoteras** [comment]
That would be helpful because it saves time

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
